tensorflow/java/src/main/java/org/tensorflow/EagerOperationBuilder.java

```diff
@@ -0,0 +1,249 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow;
+
+import java.nio.charset.Charset;
+
+/** 
+ * An {@link OperationBuilder} for building {@link Operation}s that are executed eagerly.
```

Comment:
I have a slight preference for `{@link Operation Operations}` over `{@link Operation}s` but maybe that's just me.

Reply:
Np, I'll change it